x-pack/plugins/security_solution/server/lib/entity_analytics/entity_store/task/field_retention_enrichment_task.ts

```diff
@@ -67,6 +67,7 @@ export const registerEntityStoreFieldRetentionEnrichTask = ({
       namespace,
       entityType,
       fieldHistoryLength: 10, // we are not using this value so it can be anything
+      indexPatterns: [], // we are not using this value so it can be anything
```

Comment:
@hop-dev Is fixing this

Reply:
https://github.com/elastic/kibana/pull/195961